src/main/java/Duke.java

```diff
@@ -1,10 +1,141 @@
+            } else if (inData.equals("list")){
+                for (Task task:tasks){
+                    System.out.println("Here are the tasks in your list: \n" 
+                                        + (tasks.indexOf(task) + 1) 
+                                        + "[" + task.getDescription() + "]"
+                                        + "." + "[" + task.getStatusIcon() + "] " 
+                                        + task.getDescription());
+                }
+            } else if (inData.contains("mark") && !inData.contains("unmark")){
+                int inDataIndex = Integer.parseInt(inData.substring(5));
+                inDataIndex--;
+                tasks.get(inDataIndex).markAsDone();
+                Task task = tasks.get(inDataIndex);
+                System.out.println("Nice! I've marked this task as done: \n" + "[" 
+                + task.getStatusIcon() + "] " + task.getDescription());
+            } else if (inData.contains("unmark")){
+                int inDataIndex = Integer.parseInt(inData.substring(7));
+                inDataIndex--;
+                tasks.get(inDataIndex).markAsNotDone();
+                Task task = tasks.get(inDataIndex);
+                System.out.println("OK, I've marked this task as not done yet: \n" + "[" 
+                + task.getStatusIcon() + "] " + task.getDescription());
+            } else if (inData.substring(0,4).equals("todo")){
+                Task t = new Task(inData.substring(4),false,"T");
+                tasks.add(t);
+                System.out.println("Got it. I've added this task: \n" + "["
+                + t.getDescription() + "]" + "[" + t.getStatusIcon() + "] "
+                + t.getDescription());
+                System.out.println("Now you have %2d tasks in the list.".formatted(tasks.size()));
+            } else if (inData.substring(0,8).equals("deadline")){
+                int taskEndIndex = inData.indexOf("/");
+                Task t = new Task(inData.substring(9,taskEndIndex),false,"D",
+                inData.substring(taskEndIndex+1));
+                System.out.println("Got it. I've added this task: \n" + "["
+                + t.getDescription() + "]" + "[" + t.getStatusIcon() + "] "
+                + t.getDescription() + "by: " + t.getDate());
+                System.out.println("Now you have %2d tasks in the list.".formatted(tasks.size()));
+            } else if (inData.substring(0,5).equals("event")){
+                int taskEndIndex = inData.indexOf("/");
+                Task t = new Task(inData.substring(5,taskEndIndex),false,"E",
+                inData.substring(taskEndIndex+1));
+                System.out.println("Got it. I've added this task: \n" + "["
+                + t.getDescription() + "]" + "[" + t.getStatusIcon() + "] "
+                + t.getDescription() + t.getDate());
+                System.out.println("Now you have %2d tasks in the list.".formatted(tasks.size()));
```

Comment:
repetitive print command in each case, try to print common text before and after the switch statement